### PR TITLE
feat:Published and Unpublished articles in the My articles page

### PR DESCRIPTION
**Pull request by BayramovaKonul**
Blog page shows articles which have published_at more less than current date.

### CODE REVIEW

File: blog/views.py [hunk 1]
```diff
@@ -177,11 +179,17 @@ def post(self, request):
 @login_required(login_url='login')
 def my_articles (request):
     page = request.GET.get('page')
-    all_articles = ArticleModel.objects.filter(author_id = request.user)
-    paginator = Paginator(all_articles, 3) 
+    all_articles = ArticleModel.objects.filter(author_id=request.user).order_by('published_at')
+    paginator = Paginator(all_articles, 3)
+    current_datetime = now()
+
+    for article in all_articles:
```
Comment (valiyevvali) [inline blog/views.py]: in db

File: blog/views.py [hunk 2]
```diff
@@ -177,11 +179,17 @@ def post(self, request):
 @login_required(login_url='login')
 def my_articles (request):
     page = request.GET.get('page')
-    all_articles = ArticleModel.objects.filter(author_id = request.user)
-    paginator = Paginator(all_articles, 3) 
+    all_articles = ArticleModel.objects.filter(author_id=request.user).order_by('published_at')
```
Comment (valiyevvali) [inline blog/views.py]: filter

File: blog/views.py [hunk 3]
```diff
@@ -177,11 +179,23 @@ def post(self, request):
...
+        filtered_articles = all_articles.filter(published_at__gt=current_datetime) 
+    else:
+        filtered_articles = all_articles
+        
+    paginator = Paginator(filtered_articles, 3)
+
     return render (request, 'my_articles.html', context= {
-        'all_articles':all_articles,
         'page_obj' : paginator.get_page(page),
+        'current_datetime':current_datetime,
```
Comment (valiyevvali) [inline blog/views.py:163]: why do we need this, since you are not using it in template?

File: config/settings.py [hunk 4]
```diff
@@ -52,7 +52,6 @@
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.locale.LocaleMiddleware',
-    'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.common.CommonMiddleware',
```
Comment (valiyevvali) [inline config/settings.py:60]: WHY? Deleting message middleware is not recommended

Comment (BayramovaKonul) [reply config/settings.py:60]: They were 2 message middlewares mistakenly, and I deleted one of them.

**Pull request closed without merging**